core/domain/exp_jobs_one_off_test.py

```diff
@@ -565,6 +609,272 @@ def test_nonhuman_committers_not_counted(self):
                 system_id,
                 exploration_summary.contributors_summary)
 
+    def test_no_action_is_performed_for_deleted_exploration(self):
+        """Test that no action is performed on deleted explorations."""
+
+        user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        exp_id = '100'
+        exploration = self.save_new_valid_exploration(
+            exp_id, user_a_id)
+        exp_services.delete_exploration(user_a_id, exp_id)
+
+        # Run the ExplorationContributorsSummaryOneOff job.
+        job_id = (
+            exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.create_new()) #pylint: disable=line-too-long
+        exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        with self.assertRaisesRegexp(
+            base_models.BaseModel.EntityNotFoundError,
+            'Entity for class ExpSummaryModel with id 100 not found'
+        ):
+            exp_services.get_exploration_summary_by_id(exploration.id)
+
+    def test_exploration_contributors_summary_job_produces_no_output(self):
+        """Test that ExplorationContributorsSummaryOneOff job produces
+        no output.
+        """
+
+        user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        self.save_new_valid_exploration(
+            self.EXP_ID, user_a_id, title='Exploration Title')
+
+        # Run the ExplorationContributorsSummaryOneOff job.
+        job_id = exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.create_new() # pylint: disable=line-too-long
```

Comment:
Do we really need the pylint pragma? 
Something like this would work I think.
```suggestion
        job_id = (
            exp_jobs_one_off
            .ExplorationContributorsSummaryOneOffJob.create_new())
```

Reply:
Also, if this is used elsewhere in the file, can the pragma be removed there too?

Reply:
Done.